sdk/tables/azure-data-tables/azure/data/tables/aio/_table_batch_async.py

```diff
@@ -1,604 +0,0 @@
-# -------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-# --------------------------------------------------------------------------
-from typing import Dict, Any, Optional, Union
-
-from azure.core import MatchConditions
-
-from .._common_conversion import _transform_patch_to_cosmos_post
-from .._models import UpdateMode
-from .._table_batch import EntityType, TransactionOperationType
-from .._serialize import (
-    _prepare_key,
-    _add_entity_properties,
-    _get_match_condition,
-)
-from .._generated import models
-from .._generated.aio import AzureTable
-from .._generated.aio._configuration import AzureTableConfiguration
-from .._generated._serialization import Serializer, Deserializer
-from .._generated._vendor import prep_if_match
-
-
-class TableBatchOperations(object):
```

Comment:
Why remove async batch operations?

Reply:
Turns out the entire async file was redundant, because there are no asynchronous methods. So it was just a duplicate of the sync code :)